kafka_utils/kafka_consumer_manager/commands/offset_set_timestamp.py

```diff
@@ -0,0 +1,141 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Yelp Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from __future__ import absolute_import
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import sys
+from collections import defaultdict
+from collections import OrderedDict
+from datetime import datetime
+
+import pytz
+import six
+from kafka import KafkaConsumer
+from kafka.structs import TopicPartition
+
+from .offset_manager import OffsetManagerBase
+from kafka_utils.kafka_consumer_manager.util import consumer_commit_for_times
+
+
+class OffsetSetTimestamp(OffsetManagerBase):
+
+    new_offsets_dict = defaultdict(dict)
+
+    @classmethod
+    def topics_dict(cls, string):
+        try:
+            topic, partition_timestamp = string.rsplit(".", 1)
+            partition, timestamp = partition_timestamp.split("=", 1)
+            cls.new_offsets_dict[topic][int(partition)] = int(timestamp)
+        except ValueError:
+            print(
+                "Error: Badly formatted input, please re-run command "
+                "with --help option.", file=sys.stderr
```

Comment:
instead of re-run command, maybe something like use --help or run kafka-consumer-manager with --help